zerver/lib/exceptions.py

```diff
@@ -280,6 +281,14 @@ def msg_format() -> str:
         return _("This organization has been deactivated")
 
 
+class RemoteServerDeactivatedError(AuthenticationFailedError):
+    code: ErrorCode = ErrorCode.REALM_DEACTIVATED
+
+    @staticmethod
+    def msg_format() -> str:
+        return _("This remote server has been deactivated")
```

Comment:
"The mobile push notification service registration for your server has been deactivated".